tests/miral/external_client.cpp

```diff
@@ -93,7 +93,7 @@ TEST_F(ExternalClient, default_app_env_is_as_expected)
 
 TEST_F(ExternalClient, default_app_env_x11_is_as_expected)
 {
-    if (getenv("XDG_RUNTIME_DIR") == nullptr)
+    if (access("/tmp/.X11-unix/", W_OK) == 0)
```

Comment:
This is probably ok, but it would be more principled to have a `MIR_DISABLE_X11_TESTS` CMake option in the same way that we have `MIR_DISABLE_EPOLL_REACTOR`.

Reply:
Possibly more principled, but it assumes (probably correctly) that the build environment is a predictor for the runtime environment.